src/board/system76/common/fan.c

```diff
@@ -4,14 +4,19 @@
 #include <common/debug.h>
 #include <ec/pwm.h>
 
-bool fan_max = false;
+#if SMOOTH_FANS != 0
+    const uint8_t max_jump_up = (MAX_FAN_SPEED - MIN_FAN_SPEED) / (uint8_t) SMOOTH_FANS_UP;
```

Comment:
These can be `static`.

Reply:
@crawfxrd is that in place of `const` keyword?

Reply:
In addition to it: `static const`.